.circleci/config.yml

```diff
@@ -7,11 +7,12 @@ env: &env
   environment:
     GRUNTWORK_INSTALLER_VERSION: v0.0.30
     TERRATEST_LOG_PARSER_VERSION: v0.30.4
-    MODULE_CI_VERSION: v0.29.1
-    TERRAFORM_VERSION: 0.13.5
+    MODULE_CI_VERSION: v0.31.0
```

Comment:
The latest for module ci is `v0.33.2`, and the release with TF14 is `v0.32.0`

Reply:
@marinalimeira oh bummer, maybe I used the wrong version of the script: `tf14.rb`. Let me check

Reply:
@marinalimeira should I update this? https://github.com/gruntwork-io/prototypes/blob/master/git-xargs-assets/scripts/tf14.rb#L27

Reply:
Changed to v0.33.2

Reply:
Changed to v0.33.2